src/vivarium/framework/results/interface.py

```diff
@@ -1,16 +1,14 @@
 """
-==========================
-Vivarium Results Interface
-==========================
+=================
+Results Interface
+=================
 
 This module provides a :class:`ResultsInterface <ResultsInterface>` class with
 methods to register stratifications and results producers (referred to as "observations")
 to a simulation.
 """
 
-from __future__ import annotations
```

Comment:
We already had determined that we couldn't use, e.g. `pd.Series[str]` (which requires this import to work currently), but this import itself was throwing warnings about not being able to resolve pd.Series or pd.DataFrame - it was super difficult to track this down as the culprit!